README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
This is horribly, horribly wrong - freedom of speech is a basic liberty to express opinions publicly or to call for action, by being protected from any and all attempts by *anyone* to counter this act of expression in any way, except if exigent circumstances or specific types of expression - hate speech, libel, etc.

What that means, is neither individuals, organizations, nor government can limit free speech, and that includes this CoC - if it does limit free speech in any way, it is against plenty of local and quite a few international laws. However, what is true in the XKCD you removed, is that this mostly applies to public expression - there is a long philosophical and legal debate about this, but usually the drilldown is - you can execute free speech in a local park, without limitation, however, you are not allowed to execute your free speech after breaking in to some poor guys apartment. Hence, the definition of public space also applies. 

From that and the intended audience, and purpose of the CoC we can kind of deduce that the public space paradigm is really in effect here, because otherwise it would be a private party - not open community. Both can not coexist. 

See, as an example, Universal Declaration of Human Rights by the United Nations art. 18, art. 19, European Convention on Human Rights art. 10, Act One of the Fundamental Law on the Freedom of Expression of the Kingdom of Sweden, Constitution of the Republic of Latvia art. 100., Constitution of the Kingdom of Norway art. 100, Section 2 of the Canadian Charter of Rights and Freedoms of the Constitution of Canada, etc. etc. There is plenty more.